tests/datasets/cdc/test_groupedmessage.py

```diff
@@ -0,0 +1,284 @@
+from datetime import datetime
```

Comment:
Same comment as `tests/datasets/cdc/test_groupassignee.py` (except for a different file.)